apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/index.tsx

```diff
@@ -33,6 +34,17 @@ export function EvaluationResults({
     EvaluationResultWithMetadata | undefined
   >(undefined)
   const { data: providerLog } = useProviderLog(selectedResult?.providerLogId)
+  const { data: evaluationResults } = useEvaluationResultsWithMetadata(
+    {
+      evaluationId: evaluation.id,
+      documentUuid: document.documentUuid,
+      commitUuid: commit.uuid,
+      projectId: project.id,
+    },
+    {
+      fallbackData: serverData,
+    },
+  )
```

Comment:
add the interval refetch every x seconds please

Reply:
We can manually refetch with websockets, right?

Reply:
ah you mean from actual real logs, not only from batches